src/Standards/BestIt/Sniffs/AbstractSniff.php

```diff
@@ -23,30 +23,30 @@ abstract class AbstractSniff implements Sniff
     /**
      * The used file.
      *
-     * @var File|void
+     * @var File|null
      */
-    protected $file;
+    protected File|null $file;
```

Comment:
You must initialize that property (directly or by constructor).

Otherwise you may get an uninitialized error.